src/components/pages/Offer/StocksManager/StockItem/StockItem.jsx

```diff
@@ -124,7 +110,7 @@ class StockItem extends PureComponent {
           <EventFields
             beginningDatetime={beginningDatetime}
             dispatch={dispatch}
-            readOnly={readOnly}
+            readOnly={userIsNotUpdatingStock || stockAssociatedToAllocineOffer}
```

Comment:
Pour pas utiliser offer.hasBeenProvidedByAllocine tout simplement ?
stockAssociatedToAllocineOffer augmente la complexité
```suggestion
            readOnly={userIsNotUpdatingStock || offer.hasBeenProvidedByAllocine}
```

Reply:
yes, c'est vrai.